doc/dev/crimson/crimson.rst

```diff
@@ -8,6 +8,8 @@ and storage devices, employing state-of-the-art technologies including
 DPDK and SPDK. BlueStore continues to support HDDs and slower SSDs.
 Crimson aims to be backward compatible with the classic ``ceph-osd``.
```

Comment:
this line about backward-compatibility might be misleading, suggesting that you could stop a classic osd and start a crimson one with the same store

Reply:
I've rephrased the entire paragraph